internal/builders/maven/action.yml

```diff
@@ -68,12 +68,39 @@ runs:
       shell: bash
       env:
         SLSA_OUTPUTS_ARTIFACTS_FILE: ${{ inputs.slsa-layout-file }}
+        PROJECT_ROOT: ${{ fromJson(inputs.slsa-workflow-inputs).directory }}
       run: |
+        # Ensure no directory traversal.
+        # NOTE: the actions/download-artifact Action only creates files
+        # in the workspace directory, but this may change in the future.
+        # TODO(#1893): Consolidate directory traversal checks
+        validate_path() {
+          untrusted_path=$1
+          resolved_dir=$(readlink -m "$untrusted_path")
+          wd=$(readlink -m "${GITHUB_WORKSPACE}")
+          if [[ "${wd}" != "" ]] && [[ "${resolved_dir}" != "${wd}"/* ]] && [[ "${resolved_dir}" != "${wd}" ]]; then
```

Comment:
I realize this is copied from the `secure-download-artifact` action but do we really need the `"${wd}" != ""` check? What situation does this cover? IIUC this means that if `GITHUB_WORKSPACE` is `""` for some reason this check just passes.

/cc @laurentsimon

Reply:
I dont remember why we was needed this at some point. I don't think we need it anymore

Reply:
removed it.